client/src/views/Models/components/DrilldownPanel/MetadataPane.vue

```diff
@@ -65,6 +66,42 @@
         </div>
       </template>
 
+      <template v-if="isTypeIndraAgentReferenceSet(datum)">
+        <summary :title="datum.uid">Agent(s) Reference(s)</summary>
+        <div class="metadata-content"  v-for="(reference, index) in agentsReferences" :key="index">
+           <details class="metadata" open>
+            <summary>
+              <a v-if="reference.url" :href="reference.url">
+              {{ reference.name }}
+              </a>
+            </summary>
+            <div v-if="reference.definition" class="metadata-content">
+              <p>{{ reference.definition }}</p>
+            </div>
+            <div v-else class="metadata-content">
+              No definition.
+            </div>
+          </details>
+        </div>
+      </template>
+
+      <template v-if="isTypeReactionReference(datum)">
+        <summary :title="datum.uid">Reaction Reference(s)</summary>
+            <details v-if="hasNodeEvidences" class="metadata" open>
+              <summary>Evidence ({{nodeEvidences.length}})</summary>
+              <div class="metadata-content">
+                <div class="metadata-content">
```

Comment:
Why the double `<div class="metadata-content">` ?

Reply:
I think it was just a mistake. Good catch.